kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/user/LocalUser.java

```diff
@@ -94,9 +95,19 @@ public void setFitbitUserId(String id) {
     this.externalUserId = id;
   }
 
+  @Override
+  public String getVersion() {
```

Comment:
is this the replacement of random UUID?

Reply:
Yes, basically the Auth service should set a version. It’s a string, so the Auth service van still use uuid to generate a new version ID. But it could also be a date or simple number, it depends on the Auth service.